app/_landing_pages/deck_landing.yaml

```diff
@@ -0,0 +1,225 @@
+  - row:
+    - column:
+      - type: header
+        config:
+          type: h3
+          text: "OSX"
+          align: center
+    - column:
+      - type: header
+        config:
+          type: h3
+          text: "Linux"
+          align: center
+    - column:
+      - type: header
+        config:
+          type: h3
+          text: "Windows"
+          align: center
+  - row:
+    - column:
+      - type: structured_text
+        config:
+          blocks:
+            - type: text
+              text: | 
+                1.  `brew tap kong/deck`
+                2. `brew install deck`
+    - column: 
+      - type: structured_text
+        config: 
+          blocks:
+            - type: text
+              text: |
+                1. `curl -sL https://github.com/kong/deck/releases/download/v1.38.1/deck_1.38.1_windows_amd64.tar.gz -o deck.tar.gz`
+                2. `tar -xf deck.tar.gz -C /tmp`
+                3. `sudo cp /tmp/deck /usr/local/bin/`
+  - row: 
+    - column:
+      - type: x_with_y
+        config: 
+            headers:
+              - What does decK do?
+            items:
+              - text: | 
+                  Export the current configuration from Kong Gateway
+                action:
+                  type: command
+                  config:
+                    cmd: deck
+                    args:
+                      - gateway
+                      - sync
+                      - /path/to/file.json
+              - text: | 
+                  Sync configs between Gateway and Konnect
+                action:
+                  type: command
+                  config:
+                    cmd: deck
+                    args:
+                      - gateway
+                      - ping
+              - text: | 
+                  Manage secrets
+                action:
+                  type: command
+                  config:
+                    type: consumer
+                    formats:
+                      - deck
+                  
+              - text: | 
+                  Sync configs between gateway and Konnect
+                action:
+                  type: entity_example
+                  config:
+                    type: consumer
+                    formats:
+                      - terraform
+                    data:
+                      custom_id: example-consumer-id
+                      username: example-consumer
+                      tags:
+                        - silver-tier
+
+  
+  - row:
+    - column:
+      - type: card
+        config:
+          title: decK Entities
+          cta: https://docs.konghq.com/deck/latest/reference/entities/
+    - column:
+      - type: card
+        config:
+          title: Best Practices
+          cta: https://docs.konghq.com/deck/latest/guides/best-practices/
+    - column:
+      - type: card
+        config:
+          title: Changelog
+          cta: https://github.com/kong/deck/blob/main/CHANGELOG.md/
+  - row:
```

Comment:
I like the idea of splitting the FAQs on a landing page into columns in principle, but the lengths of these answers vary wildly, so we end up with:

![Screenshot 2024-08-07 at 2 03 52 PM](https://github.com/user-attachments/assets/97e936e8-87ff-4abb-aa3e-0a8c87924df4)

Either we should stick to one column width, collapsible FAQs (which is good for consistency anyway), or some of these answers are too long and need to be somewhere else (eg "Kong already has built-in declarative configuration, do I still need decK?") - except I don't know where the somewhere else is.

For now, I vote for one column, and collapsible sections, for consistency with how we're implementing this elsewhere.